src/macrofier/engine.mjs

```diff
@@ -328,7 +329,7 @@ const generateMacros = (obj, templates, languages, options = {}) => {
   const allMethodsArray = generateMethods(obj, examples, templates)
   const methodsArray = allMethodsArray.filter(m => !m.event && (!options.hideExcluded || !m.excluded))
   const eventsArray = allMethodsArray.filter(m => m.event && (!options.hideExcluded || !m.excluded))
-  const declarationsArray = allMethodsArray.filter(m => m.declaration)
+  const declarationsArray = allMethodsArray.filter(m => m.declaration && (!options.hideExcluded || !m.excluded))
```

Comment:
in native case, it is generating declaration alone with out implementation. In case of Device Version property, it generate method signature for header, but nothing for implementation since that property type tag is exclude-from-sdk